### PR TITLE
Refactor variable names for Google Cloud Provider

### PR DESCRIPTION
**Pull request by ilausuch**
- Related ticket: https://progress.opensuse.org/issues/98379
- Verification run: https://openqa.suse.de/tests/7843257

Dependency with https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/538 (both has to be merged at the same time)

### CODE REVIEW

File: lib/publiccloud/basetest.pm [hunk 1]
```diff
@@ -71,27 +71,27 @@ sub provider_factory {
...
+                storage_name => get_var('PUBLIC_CLOUD_GCP_STORAGE', 'openqa-storage'),
                 username => get_var('PUBLIC_CLOUD_USER', 'susetest')
             );
         }
         elsif ($args{service} eq 'GCE') {
             $provider = publiccloud::gce->new(
-                account => get_var('PUBLIC_CLOUD_ACCOUNT'),
-                service_acount_name => get_var('PUBLIC_CLOUD_SERVICE_ACCOUNT'),
-                project_id => get_var('PUBLIC_CLOUD_PROJECT_ID'),
+                account => get_var('PUBLIC_CLOUD_GCEACCOUNT'),
```
Comment (grisu48) [inline lib/publiccloud/basetest.pm:87]:
```suggestion
                account => get_var('PUBLIC_CLOUD_GCP_ACCOUNT'),
```

Comment (ilausuch) [reply lib/publiccloud/basetest.pm]: done, thanks

File: lib/publiccloud/basetest.pm [hunk 2]
```diff
@@ -71,27 +71,27 @@ sub provider_factory {
...
+                storage_name => get_var('PUBLIC_CLOUD_GCP_STORAGE', 'openqa-storage'),
                 username => get_var('PUBLIC_CLOUD_USER', 'susetest')
             );
         }
         elsif ($args{service} eq 'GCE') {
             $provider = publiccloud::gce->new(
-                account => get_var('PUBLIC_CLOUD_ACCOUNT'),
-                service_acount_name => get_var('PUBLIC_CLOUD_SERVICE_ACCOUNT'),
-                project_id => get_var('PUBLIC_CLOUD_PROJECT_ID'),
+                account => get_var('PUBLIC_CLOUD_GCP_ACCOUNT'),
```
Comment (pdostal) [inline lib/publiccloud/basetest.pm:87]: Here we're in *GCE* but the variables contain *GCP* plus on line 72 we can also see *GCR*.
Honestly don't follow this any more. What about `PUBLIC_CLOUD_GOOGLE_` instead?

Comment (ilausuch) [reply lib/publiccloud/basetest.pm:87]: ok... I agree that in this way is more clear

Comment (ilausuch) [reply lib/publiccloud/basetest.pm]: Done

**Review by pdostal — APPROVED** [review]
Thank you, Ivan! :) Let's maybe also notify @asmorodskyi ?

File: lib/publiccloud/basetest.pm [hunk 3]
```diff
@@ -71,27 +71,27 @@ sub provider_factory {
         $args{service} //= 'GCE';
         if ($args{service} eq 'GCR') {
             $provider = publiccloud::gcr->new(
-                account => get_var('PUBLIC_CLOUD_ACCOUNT'),
-                service_acount_name => get_var('PUBLIC_CLOUD_SERVICE_ACCOUNT'),
-                project_id => get_var('PUBLIC_CLOUD_PROJECT_ID'),
+                account => get_var('PUBLIC_CLOUD_GOOLE_ACCOUNT'),
```
Comment (jlausuch) [inline lib/publiccloud/basetest.pm:74]:
```suggestion
                account => get_var('PUBLIC_CLOUD_GOOGLE_ACCOUNT'),
```

File: lib/publiccloud/basetest.pm [hunk 4]
```diff
@@ -71,27 +71,27 @@ sub provider_factory {
...
+                storage_name => get_var('PUBLIC_CLOUD_GOOGLE_STORAGE', 'openqa-storage'),
                 username => get_var('PUBLIC_CLOUD_USER', 'susetest')
             );
         }
         elsif ($args{service} eq 'GCE') {
             $provider = publiccloud::gce->new(
-                account => get_var('PUBLIC_CLOUD_ACCOUNT'),
-                service_acount_name => get_var('PUBLIC_CLOUD_SERVICE_ACCOUNT'),
-                project_id => get_var('PUBLIC_CLOUD_PROJECT_ID'),
+                account => get_var('PUBLIC_CLOUD_GOOLE_ACCOUNT'),
```
Comment (jlausuch) [inline lib/publiccloud/basetest.pm:87]:
```suggestion
                account => get_var('PUBLIC_CLOUD_GOOGLE_ACCOUNT'),
```

File: variables.md [hunk 5]
```diff
@@ -165,19 +165,20 @@ PUBLIC_CLOUD_LTP | boolean | false | If set, the run_ltp test module is added to
...
-PUBLIC_CLOUD_PROJECT_ID | string | "" | GCP only, used to specify the project id.
+PUBLIC_CLOUD_GOOGLE_PROJECT_ID | string | "" | GCP only, used to specify the project id.
 PUBLIC_CLOUD_PROVIDER | string | "" | The type of the CSP (e.g. AZURE, EC2, GCE).
 PUBLIC_CLOUD_QAM | boolean | false | Previously used to recognize maintenance jobs - now deprecated - should be removed.
 PUBLIC_CLOUD_REBOOT_TIMEOUT | integer | 600 | Number of seconds we wait for instance to reboot.
 PUBLIC_CLOUD_REGION | string | "" | The region to use. (default-azure: westeurope, default-ec2: eu-central-1, default-gcp: europe-west1).
 PUBLIC_CLOUD_RESOURCE_NAME | string | "openqa-vm" | The name we use when creating our VM.
 PUBLIC_CLOUD_SKIP_MU | boolean | false | Debug variable used to run test without maintenance updates repository being applied.
-PUBLIC_CLOUD_SERVICE_ACCOUNT | string | "" | GCE only, used to specify the service account.
+PUBLIC_CLOUD_GOOLE_ACCOUNT | string | "" | GCE only, used to specify the account id.
```
Comment (jlausuch) [inline variables.md:175]:
```suggestion
PUBLIC_CLOUD_GOOGLE_ACCOUNT | string | "" | GCE only, used to specify the account id.
```

**Review by jlausuch — CHANGES_REQUESTED** [review]
s/GOOLE/GOOGLE

**Review by asmorodskyi — APPROVED** [review]
LGTM ( after fixing GOOLE ofc :) )

File: variables.md (hunk 5 above)
Comment (ilausuch) [reply variables.md]: Done

**Comment by ilausuch** [comment]
> s/GOOLE/GOOGLE

Done in all cases

**Review by grisu48 — APPROVED** [review]
LGTM

**Pull request merged**